.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: ruby
 cache: bundler
 rvm:
-- 1.9.3
-- 2.0.0
-- 2.1
-- 2.2
+  - 1.9.3
+  - 2.0.0
+  - 2.1
+  - 2.2
+  - jruby-head
```

Comment:
Can we keep up with this? I think we should lock to a set of versions if we can.